pom.xml

```diff
@@ -137,8 +137,8 @@
                 <version>${assertj-core.version}</version>
             </dependency>
             <dependency>
-                <groupId>javax.validation</groupId>
-                <artifactId>validation-api</artifactId>
+                <groupId>jakarta.validation</groupId>
+                <artifactId>jakarta.validation-api</artifactId>
```

Comment:
same here I would go for `validation-api`

Reply:
Same answer.